src/libraries/System.Formats.Asn1/pkg/System.Formats.Asn1.pkgproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Formats.Asn1.csproj">
```

Comment:
Just dropping a note here, I believe it is kind of our intention now to try not shipping reference assembly assets on new packages unless for some reason we actually need them, so this is a good candidate for not having them and just having the src project. cc: @ericstj

Reply:
If you end up wanting to do this which I think might be beneficial, all you have to do is to remove the ProjectReference to the ref project, and just keep the src one. If you do that, you'll have to add the metadata of SupportedFrameworks you had on the ref one into the src one.

Reply:
It looks like dropping the ref node and leaving src as-is makes the build fail with the "too many things are supported" errors.  This works, but I'm not sure if it's right.  Care to comment on it before I commit?

```diff
  <ItemGroup>
-    <ProjectReference Include="..\ref\System.Formats.Asn1.csproj">
+    <ProjectReference Include="..\src\System.Formats.Asn1.csproj">
      <SupportedFramework>net461;netcoreapp2.0;uap10.0.16299;$(AllXamarinFrameworks)</SupportedFramework>
    </ProjectReference>
-    <ProjectReference Include="..\src\System.Formats.Asn1.csproj" />
  </ItemGroup>
```

Reply:
yup that should work and it is fine. That is what I meant above when I said:
> If you do that, you'll have to add the metadata of SupportedFrameworks you had on the ref one into the src one.

Reply:
Oh, right.  Yep.  I can read.  ::whistles innocently::